test/extended/util/disruption/controlplane/controlplane.go

```diff
@@ -4,33 +4,54 @@ import (
 	"context"
 	"time"
 
+	"k8s.io/client-go/rest"
+
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/upgrades"
 
 	"github.com/openshift/origin/pkg/monitor"
 	"github.com/openshift/origin/test/extended/util/disruption"
 )
 
-// AvailableTest tests that the control plane remains is available
-// before and after a cluster upgrade.
-type AvailableTest struct {
+// KubeAvailableTest tests that the Kubernetes control plane remains available during and after a cluster upgrade.
+type KubeAvailableTest struct {
+	availableTest
+}
+
+func (KubeAvailableTest) Name() string        { return "kube-api-available" }
```

Comment:
`kubernetes-api-available` just to be clear

Reply:
Oh, this is your backport.  It's fine.

Reply:
updated. Now you have to retag this and https://github.com/openshift/origin/pull/24844